assets/wizards/googleAdManager/views/editAdSlotsScreen.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * New/Edit Ad Slot Screen.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Card,
+	FormattedHeader,
+	Button,
+	TextControl,
+	withWizardScreen,
+} from '../../../components/src';
+
+/**
+ * New/Edit Ad Slot Screen.
+ */
+class EditAdSlotScreen extends Component {
+	/**
+	 * Handle an update to an ad slot field.
+	 *
+	 * @param string key Ad Slot field
+	 * @param mixed  value New value for field
+	 *
+	 */
+	handleOnChange( key, value ) {
+		const { adSlot, onChange } = this.props;
+		adSlot[ key ] = value;
+		onChange( adSlot );
+	}
+
+	/**
+	 * Render.
+	 */
+	render() {
+		const { adSlot, onClickSave } = this.props;
+		const { id, name, code } = adSlot;
+
+		const editing_existing_ad_slot = !! id;
+		return (
+			<div className="newspack-edit-ad-slot-screen">
```

Comment:
I don't think this `div` is needed. You could replace with a `Fragment` to simplify.